cosmo_tester/resources/attributes.yaml

```diff
@@ -4,6 +4,12 @@ cloudify_username: admin
 cloudify_password: admin
 cloudify_tenant: default_tenant
 
+# This should be set to true if testing community images.
+# This isn't detected by the framework, as doing so would require either some
+# heuristics or deploying managers for each test before determining whether it
+# could be skipped, both of which have a good chance of wasting a lot of time.
+image_is_community: false
```

Comment:
What if we want to test via Jenkins? Shouldn't it be possible to set this via something else, like an env var, from the Jenkins?